src/components/forms/EntryForm.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import axios from 'axios';
+import { FormContainer } from '../shared/FormContainer'
+import { FormFieldLabel } from '../shared/FormFieldLabel'
+import { useForm } from "react-hook-form";
+
+export const EntryForm = (props) => {
+    const { register, handleSubmit, watch, formState: { errors } } = useForm();
+    const onSubmit = (data) => {}
+    return (
+    <div>
```

Comment:
Minor comment, I'd change the highest level <div> into a fragment tag (<></>) instead for best practice. This way, the hierarchy of the compiled page will be clean.